scripts/set-pool-data-xbtc.js

```diff
@@ -12,12 +12,10 @@ const txOptions = {
   functionArgs: [
     tx.contractPrincipalCV(CONTRACT_ADDRESS, 'arkadiko-stake-pool-wstx-xbtc-v1-1'),
     tx.stringAsciiCV('wSTX-xBTC LP'),
+    tx.uintCV(55200), // completely disable
```

Comment:
To be fully correct you would also need to set "deactivated-rewards-per-block" to the amount of rewards per block on block "deactivated-block". But it's not really needed as most rewards are already added.